src/app/ui/export_file_window.cpp

```diff
@@ -123,7 +123,9 @@ void ExportFileWindow::savePref()
 
 std::string ExportFileWindow::outputFilenameValue() const
 {
-  return base::get_absolute_path(base::join_path(m_outputPath, m_outputFilename));
+  return (base::is_absolute_path(m_outputFilename) ?
+            base::get_absolute_path(m_outputFilename) :
+            base::get_absolute_path(base::join_path(m_outputPath, m_outputFilename)));
```

Comment:
In this case I'd prefer to comment about why we are calling `get_absolute_path()` even when the condition says that the path is already absolute (which is a confusing), this is an example:
```suggestion
  if (base::is_absolute_path(m_outputFilename)) {
    // Although the output filename already contains an absolute path,
    // calling get_absolute_path() normalizes it (removes any extra component)
    // which is useful to hit the same recent filename/path.
    return base::get_absolute_path(m_outputFilename);
  }
  return base::get_absolute_path(base::join_path(m_outputPath, m_outputFilename)));
```